[LegacyBridge] Clearify scope of support for Legacy bridge on eZ Platform 1.x and 2.x (#901)
ezsystems/developer-documentation: docs/index.md

> # eZ Platform developer documentation

Reviewer: ```suggestion
 # eZ Platform and eZ Platform Enterprise developer documentation
```

> Welcome to eZ Platform & eZ Platform Enterprise developer documentation.

Reviewer: ```suggestion
```

> Welcome to eZ Platform developer documentation, unless otherwise noted this documentation covers pure eZ Platform and eZ Platform Enterpise features only. For other sofware & solutions from eZ Systems, see "Further documentation" below.

Author: Get that. What about having something introducing what it _is_, ala:
```suggestion
Welcome to eZ Platform developer documentation, covering eZ Platform and eZ Platform Enterprise features.
```

> Welcome to eZ Platform developer documentation, unless otherwise noted this documentation covers pure eZ Platform and eZ Platform Enterpise features only. For other sofware & solutions from eZ Systems, see "Further documentation" below.

Reviewer: Or maybe even better, add Enterprise to the page title (where it should have been mentioned anyway)? It will be even more conspicuous that way.

> Welcome to eZ Platform developer documentation, unless otherwise noted this documentation covers pure eZ Platform and eZ Platform Enterpise features only. For other sofware & solutions from eZ Systems, see "Further documentation" below.

Author: We can, but it's not clarifying the overall scope of the I had in mind (ref PR description).

> Welcome to eZ Platform developer documentation, unless otherwise noted this documentation covers pure eZ Platform and eZ Platform Enterpise features only. For other sofware & solutions from eZ Systems, see "Further documentation" below.

Reviewer: I'm not clear on how `covering eZ Platform and eZ Platform Enterprise features` explains the scope better than `eZ Platform and eZ Enterprise developer documentation`

> Welcome to eZ Platform developer documentation, unless otherwise noted this documentation covers pure eZ Platform and eZ Platform Enterpise features only. For other sofware & solutions from eZ Systems, see "Further documentation" below.

Author: So what I might need to solve my issue is to create a seperate page for legacy bridge to example the limitations of it. So I can drop this and do that instead, ok?

> Welcome to eZ Platform developer documentation, unless otherwise noted this documentation covers pure eZ Platform and eZ Platform Enterpise features only. For other sofware & solutions from eZ Systems, see "Further documentation" below.

Reviewer: That would be preferable from my POV, yes. I think somewhere in "Migrating to eZ Platform" would be a good place. Then we could link to it from relevant places in requirements, and from this front page as well.

> Welcome to eZ Platform developer documentation, unless otherwise noted this documentation covers pure eZ Platform and eZ Platform Enterpise features only. For other sofware & solutions from eZ Systems, see "Further documentation" below.

Author: ```suggestion
Welcome to eZ Platform & eZ Platform Enterprise developer documentation.
```

  <div class="card-header">Further documentation</div>
  <ul class="list-group list-group-flush">
      <li class="list-group-item"><a href="https://doc.ezplatform.com/projects/userguide/en/latest/" class="card-link">User Guide</a></li>
      <li class="list-group-item"><a href="https://doc.ezplatform.com/projects/ezpersonalization/en/latest/" class="card-link">eZ Personalization</a></li>
    <li class="list-group-item"><a href="https://doc.ez.no/display/EZP/eZ+Publish+5.x+Developer+Documentation" class="card-link">eZ Publish 5</a></li>

>     <li class="list-group-item"><a href="https://github.com/ezsystems/LegacyBridge/blob/master/INSTALL.md" class="card-link">Legacy Bridge</a></li>

Author: This should have linked to the new page instead 🙃
